otx/algorithms/segmentation/adapters/mmseg/models/utils/normalize.py

```diff
@@ -1,31 +1,40 @@
-# Copyright (C) 2022 Intel Corporation
+"""Normalization."""
+# Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
 
 import torch
-import torch.nn as nn
 import torch.nn.functional as F
+from torch import nn
+
+# pylint: disable=abstract-method, unused-argument, invalid-name
```

Comment:
Please disabling pylint as minimum as possible. How about setting in on necessary functions or classes?

Reply:
Let me do that later. I leave them as is because I have no idea yet about renaming. I need to see them anyway when I debug mmseg fcnhead.